src/Behat/Behat/Output/Statistics/Statistics.php

```diff
@@ -1,239 +1,108 @@
 <?php
-
-/*
- * This file is part of the Behat.
- * (c) Konstantin Kudryashov <ever.zet@gmail.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
 namespace Behat\Behat\Output\Statistics;
 
-use Behat\Behat\Tester\Result\StepResult;
-use Behat\Testwork\Counter\Memory;
 use Behat\Testwork\Counter\Timer;
-use Behat\Testwork\Tester\Result\TestResult;
-use Behat\Testwork\Tester\Result\TestResults;
+use Behat\Testwork\Counter\Memory;
```

Comment:
Can you keep the Memory use statement before the Timer one (alphabetical sorting) ?